### ISSUE TITLE
AberrantExpression_pipeline_OUTRIDER_results_R crashes when RNA_ID is integer

### ISSUE THREAD

**Issue by schance995** [gagneurlab/drop#348]
If the RNA_ID is an integer then `Scripts/AberrantExpression/pipeline/OUTRIDER/results.R` fails with the following:
```
Error in bmerge(i, x, leftcols, rightcols, roll, rollends, nomatch, mult,  : 
  Incompatible join types: x.RNA_ID (integer) and i.sampleID (character)
Calls: add_HPO_cols ... merge -> merge.data.table -> [ -> [.data.table -> bmerge
Execution halted
```
This is because when the tsv file is read, the type of RNA_ID is inferred to be integer instead of character.
Could an explicit cast of RNA_ID and DNA_ID to character type be added to DROP? Or at least the documentation could require that RNA_ID and DNA_ID are strings instead of integers (eg RNA1234 instead of 1234).

**Comment by nickhsmith** [comment]
Yes this is a good find. We will push this fix to allow for integer sampleIDs to be read as strings.

**Comment by vyepez88** [comment]
thanks also for noticing this! We've now added checks when reading the sample annotation in different scripts to ensure the ids are read as characters. You can find such fix in this commit: 02ac56a39e42dc7a0bff96113229d25913d698d1 (among others). The fixes will be integrated in the master branch in the next release.
